index.ts

```diff
@@ -8,6 +8,9 @@ import { Temporal } from '@js-temporal/polyfill';
 
 import { BASELINE_LOW_TO_HIGH_DURATION } from 'compute-baseline';
 
+// The longest name allowed, to allow for compact display.
+const nameMaxLength = 50;
```

Comment:
I like the fact that we enfore a max limit. I imagine this could be a useful guarantee to consumers of the data.
I don't know about picking 50 though. It feels a tad short and I'm guessing we might need to up that limit at some point in the future.
How about picking 100 instead? It feels more manageable on the long run, and would still play the role of communicating **a** limit to consumers.

Reply:
My vote goes to 80.

Reply:
80 it is! I'll also back out the name changes that aren't needed then.